webapp/src/translations/languages/en.json

```diff
@@ -360,7 +360,7 @@
         "yourLiquidity": "Your liquidity",
         "yourLiquidityInfo": "You can now liquidity mine for potential high yields. Pick a pool pair below, supply liquidity to power the DEX (Decentralized Exchange) to start earning fees and annual returns. You may withdraw at any time.",
         "noLiquidity": "No liquidity found. Would you like to add liquidity to the pool?",
-        "loading": "Loading…",
+        "loading": "Loading ...",
```

Comment:
"Loading..." should remain as "Loading...", no `space`. However, the `.` used is correct.

Reply:
You are right. I did some research and removed the space before the ellipsis.